parameter_files/transaction.txt

```diff
@@ -1,11 +1,3 @@
-Bob Loblaw
-0002
-0005     3
-0006     7
-0001     10
-CASH $0020.00
-
-Mary Sue
-0004     2
-0009     15
-CREDIT 12345
\ No newline at end of file
+Bob Loblaw    0002 1 0005 3 0006 7 0001 10    CASH $0050.00
```

Comment:
This new format for the `transaction.txt` doesn't follow the specification on pg. 35 of the reader